packages/js/product-editor/src/utils/record-product-event.ts

```diff
@@ -34,9 +37,9 @@ export function recordProductEvent( eventName: string, product: Product ) {
 
 	recordEvent( eventName, {
 		attributes: attributes.length ? 'yes' : 'no',
-		categories: categories.length ? 'yes' : 'no',
-		cross_sells: cross_sell_ids.length ? 'yes' : 'no',
-		description: description.length ? 'yes' : 'no',
+		categories: categories?.length ? 'yes' : 'no',
+		cross_sells: cross_sell_ids?.length ? 'yes' : 'no',
+		description: description?.length ? 'yes' : 'no',
```

Comment:
Is it clear to mark these as `no` when the product type never has them? I guess it depends on how the tracks events are used for analysis.

And, do any of these get inherited from the parent variable product? If so, we really should record the inherited value.

Reply:
Yeah this needs to be cleaned up!
For the sake of getting something working quickly, I resorted to the above. We did have to update this to be more flexible with other product types.